reduction/imaging_parameters.py

```diff
@@ -58,11 +58,140 @@
                                'scales': [0,3,9,27],
                               },
 }
+
+
+# for the first image, and first selfcal
+# to make the first image that is to be used for the first iteration of selfcal
+# the cleaning is better to be relatively shallow (e.g., 4 rms)
+# remove the affix "_firstim" before you make the first image, followed by the first iteration of selfcal
+imaging_parameters_nondefault_firstim = {
```

Comment:
This won't work.  It is important that people _not_ modify this file when running the code.

You're suggesting here that the first-pass clean needs significantly different parameters than subsequent iterations?  That's OK, but we need to do it a different (i.e., _fully_ automatic) way.  Editing source code files is not allowed.

Reply:
Instead of what you had before, you can do something like:
```
    'G353.41_B3_7M12M_robust-2': {'threshold': {0: '1.8mJy', 1:'4.4mJy', 2:'4.4mJy', },
                             'scales': [0,3,9,27],
                            },
```
which will change the threshold at each iteration but leave the scales the same for all iterations.  You need to have an entry (a number) for each iteration, though.

I forgot to document this...